project-base/storefront/components/Blocks/Login/loginFormMeta.ts

```diff
@@ -7,15 +9,21 @@ import { useOnFinishHydrationDefaultValuesPrefill } from 'utils/forms/useOnFinis
 import { useShopsysForm } from 'utils/forms/useShopsysForm';
 import * as Yup from 'yup';
 
+export const testSchema = Yup.object().shape<Record<keyof LoginFormType, any>>({
+    email: Yup.string().required('This field is required').email('This value is not a valid email'),
+    password: Yup.string().required('This field is required'),
+});
+
```

Comment:
This is probably forgotten, or do we need it?

Reply:
fixed :)